composer/trainer/trainer.py

```diff
@@ -782,9 +812,7 @@ def __init__(
         # Save Checkpoint
         save_folder: Optional[str] = None,
         save_filename: str = 'ep{epoch}-ba{batch}-rank{rank}.pt',
-        save_artifact_name: str = '{run_name}/checkpoints/ep{epoch}-ba{batch}-rank{rank}',
```

Comment:
same comment as above about losing the ability to name local/remote differently. Maybe I'm just forgetting what we decided, but I thought we were just going to name these args differently like `save_remote_filename` or whatever

Reply:
For these ones, we decided that if you specify `s3://{bucket}{path}`, then the artifact name will be {path} {filename} because the actual filename is the same for both; artifact name just has the parent path prefixed to it, but we can extract the parent path from `save_folder`